### PR TITLE
add required sources to zon paths

### PR DESCRIPTION
**Pull request by dasimmet**
##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: build.zig.zon [hunk 1]
```diff
@@ -2,8 +2,16 @@
     .name = "openssl",
     .version = "3.4.0",
     .paths = .{
+        "*.h",
```
Comment (kassane) [inline build.zig.zon]: What headers are you referring to?

File: build.zig.zon [hunk 2]
```diff
@@ -2,8 +2,16 @@
...
         "build.zig",
+        "crypto",
+        "include_gen",
+        "include/crypto",
+        "include/internal",
+        "include/openssl",
```
Comment (kassane) [inline build.zig.zon]: Would it be worth adding just base include dir?

**Review by kassane — CHANGES_REQUESTED** [review]

File: build.zig.zon (hunk 1 above)
Comment (dasimmet) [reply build.zig.zon]: sorry, i added this by trial and error accidentally

File: build.zig.zon (hunk 2 above)
Comment (dasimmet) [reply build.zig.zon]: done

**Review by kassane — APPROVED** [review]

**Comment by kassane** [comment]
>     * [ ]  documentation is added or updated
>     * [ ]  tests are added or updated

Hi, @dasimmet 

Excuse me! I don't understand this check list yet.
Do you have anything to add or done?

**Comment by dasimmet** [comment]
Hi,

No and no, it was just autogenerated issue text because of the files from
upstream

**Comment by kassane** [comment]
Ok. Thanks for contribution!

**Pull request merged**